test/concurrent.js

```diff
@@ -1,8 +1,8 @@
 import {serial as test} from 'ava';
 import delay from 'delay';
-import Listr from '..';
```

Comment:
Why did you edit these files? They're out of the scope of this PR anyways, and the imports were fine the way they were.

Reply:
The changes come from `xo --fix` command. I remove them in my next commit.